client/driver/plugins.go

```diff
@@ -14,15 +17,18 @@ var HandshakeConfig = plugin.HandshakeConfig{
 	MagicCookieValue: "e4327c2e01eabfd75a8a67adb114fb34a757d57eee7728d857a8cec6e91a7255",
 }
 
-func GetPluginMap(w io.Writer) map[string]plugin.Plugin {
+func GetPluginMap(w io.Writer, logLevel string) map[string]plugin.Plugin {
 	e := new(ExecutorPlugin)
-	e.logger = log.New(w, "", log.LstdFlags)
+	filter := &logutils.LevelFilter{
+		Levels:   []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERR"},
+		MinLevel: logutils.LogLevel(strings.ToUpper(logLevel)),
+		Writer:   ioutil.Discard,
+	}
+
+	e.logger = log.New(filter, "", log.LstdFlags)
```

Comment:
`log.New(logOutput, "", log.LstdFlags|log.Lmicroseconds)` to be like the client. Easier to correlate stuff when debugging